react/react dependency installer/8f8bd450-3f2b-45a5-abcc-12b633899c77.json

```diff
@@ -0,0 +1,127 @@
+{
+  "name": "react/React Dependency Installer",
+  "description": "Installs dependencies defined by the user for the react installer node. Node uses npm to install the dependencies on the same application path as the react application.",
+  "interface": {
+    "inputs": {
+      "1fc459fc-545e-4675-9916-b94bcab3563c": {
+        "type": "string[]",
+        "name": "dependencies",
+        "description": "Array of npm dependencies to install.\n\neg.\n\n[\"a\",\"b\"]"
+      },
+      "9484b15d-ca86-4d63-b07f-cb4f8f21e88b": {
+        "type": "string",
+        "name": "projectPath",
+        "description": "[Inherited from port `projectPath` of `syncer`] \nReceives full file path of react application."
+      },
+      "6668cb7f-874b-4c24-ae30-89fe2e24bad1": {
+        "type": "any",
+        "name": "start",
+        "description": "[Inherited from port `start` of `syncer`] \nReceives individual item for syncing."
+      }
+    },
+    "outputs": {
+      "82d5b775-4619-466e-bb33-52c9b630500b": {
+        "type": "typeof `stdout` of `react dependency installer`",
+        "name": "stdout"
+      },
+      "9c6331a7-5dde-4bb4-a12d-f4a7b1b02ce1": {
+        "type": "typeof `stderr` of `react dependency installer`",
+        "name": "stderr"
+      },
+      "9c820c27-01f9-4a84-b2d3-a112e05dbb72": {
+        "type": "typeof `error` of `react dependency installer`",
+        "name": "error"
+      },
+      "4655960f-34c8-41ff-b9e0-3fe400b97c5d": {
+        "type": "typeof `bounced` of `react dependency installer`",
+        "name": "bounced"
+      }
+    }
+  },
+  "implementations": {
+    "cranq": {
+      "nodes": {
+        "1cd55979-d276-4251-8602-b1e7584641e6": {
+          "coordinates": [
+            95,
+            -55
+          ],
+          "name": "Dependency Installer",
```

Comment:
By convention prototype names are nouns and start with capital letter whereas node (aka instance of prototypes) names are verbs (if possible) and start with lowercase letter.
The intentions is to have a kind of readable sentence describing the activity if we read the names of nodes following the connections.
In this case I'd suggest: `install dependencies`

Reply:
changed as suggested